clang/test/Headers/ms-intrin.cpp

```diff
@@ -36,6 +36,28 @@ typedef __SIZE_TYPE__ size_t;
 
 #include <intrin.h>
 
+#ifdef __ARM_ACLE
+// arm_acle.h needs some stdint types, but -ffreestanding prevents us from
```

Comment:
Shouldn't that be fixed in arm_acle.h itself so it includes the headers with the types it needs?

Reply:
`arm_acle.h` already includes the correct header, `stdint.h`, the problem here is this test uses a custom `stdint.h` in the `Inputs/include` directory that doesn't have the required types, a better fix is to just add this code to that custom `stdint.h`, I'll do that in the next commit, thanks.